openvpn/config.sls

```diff
@@ -89,11 +89,17 @@ openvpn_config_{{ type }}_{{ name }}_passwd_file:
       - service: {{ service_id }}
 {% endif %}
 
-{% if config.tls_auth is defined and config.ta_content is defined %}
+{% if config.ta_content is defined %}
 # Deploy {{ type }} {{ name }} TLS key file
+  {% if config.tls_auth is defined and config.tls_crypt is not defined %}
```

Comment:
What happens if both `tls_crypt` and `tls_auth` are defined? I guess there'll be a cryptic SLS parser error.
IMHO it's better to let `tls_crypt` overrule `tls_auth`. Do you agree?

Alex